stories/Toggles.js

```diff
@@ -1,203 +1,203 @@
-      { title: 'привет', label: 'hello_russian' },
-      { title: '你好', label: 'hello_chinese' }
-    ]} onChange={(label, index) => {
-      alert(`clicked "${label}" at ${index} index`);
-    }} type="horizontal"/>
-  ))
-  .add('selected 2nd by default - horizontal', () => (
-    <Toggles selectedLabel="world" items={[
-      { title: 'Hello', label: 'hello' },
-      { title: 'World', label: 'world' },
-      { title: 'Bye bye', label: 'bye' },
-    ]} type="horizontal"/>
-  ))
-  .add('with label state management - horizontal', () => (
-    <TogglesWithLabelState
-      items={[
-        { title: 'Hello', label: 'hello' },
-        { title: 'World', label: 'world', disabled: true },
-        { title: 'Bye bye', label: 'bye' },
-      ]}
-      type="horizontal"/>
-  ))
-  .add('with index state management - horizontal', () => (
-    <TogglesWithIndexState
-      items={[
-        { title: 'Hello', label: 'hello' },
-        { title: 'World', label: 'world' },
-        { title: 'Bye bye', label: 'bye' },
-      ]}
-      type="horizontal"/>
-  ))
-
-  .add('with onClick - vertical', () => (
-    <Toggles selectedLabel="hello" items={[
-      { title: 'Hello', label: 'hello' },
-      { title: 'World', label: 'world' },
-      { title: 'Bye bye', label: 'bye' },
-      { title: 'привет', label: 'hello_russian' },
-      { title: '你好', label: 'hello_chinese' }
-    ]} onChange={(label, index) => {
-      alert(`clicked "${label}" at ${index} index`);
-    }} type="vertical"/>
-  ))
-  .add('selected 2nd by default - vertical', () => (
-    <Toggles selectedLabel="world" items={[
-      { title: 'Hello', label: 'hello' },
-      { title: 'World', label: 'world' },
-      { title: 'Bye bye', label: 'bye' },
-    ]} type="vertical"/>
-  ))
-  .add('with label state management - vertical', () => (
-    <TogglesWithLabelState
-      items={[
-        { title: 'Hello', label: 'hello' },
-        { title: 'World', label: 'world', disabled: true },
-        { title: 'Bye bye', label: 'bye' },
-      ]}
-      type="vertical"/>
-  ))
-  .add('with index state management - vertical', () => (
-    <TogglesWithIndexState
-      items={[
-        { title: 'Hello', label: 'hello' },
-        { title: 'World', label: 'world' },
-        { title: 'Bye bye', label: 'bye' },
-      ]}
-      type="vertical"/>
-  ))
-  .add('with disabled - horizontal', () => (
-    <Toggles selectedLabel="hello"
-      items={[
-        { title: 'Hello', label: 'hello' },
-        { title: 'World', label: 'world', disabled: true },
-        { title: 'Bye bye', label: 'bye' },
-      ]}
-      type="horizontal"/>
-  ))
-  .add('with disabled - vertical', () => (
-    <Toggles selectedLabel="hello"
-      items={[
-        { title: 'Hello', label: 'hello' },
-        { title: 'World', label: 'world' },
-        { title: 'Bye bye', label: 'bye', disabled: true },
-      ]}
-      type="vertical"/>
-  ));
-
+//
+//
+// import React from 'react';
```

Comment:
Why did you comment out the Toggle story?

Reply:
When I pull your branch and open plasma in editor, I don't see this change. Odd.

Reply:
It's the _Toggles_ story. I don't know why that exists, but it doesn't work.
I think it wasn't included in the `index.js` previously.

I can delete the file, but I wasn't sure I should.

Reply:
It can be deleted. It looks like it is referencing a component that does not exist. 

```
import Toggles from '../src/components/Toggles/Toggles';
```